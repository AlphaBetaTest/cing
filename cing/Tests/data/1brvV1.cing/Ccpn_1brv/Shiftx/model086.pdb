REMARK PDB file of molecule Ccpn_1brv                                 
ATOM      1  N   VAL A 171       5.269  -0.948   4.643  1.00  0.00              
ATOM      2  CA  VAL A 171       4.594  -0.160   3.585  1.00  0.00              
ATOM      3  HA  VAL A 171       4.277   0.819   3.992  1.00  0.00              
ATOM      4  CB  VAL A 171       5.679   0.154   2.497  1.00  0.00              
ATOM      5  HB  VAL A 171       5.116   0.635   1.698  1.00  0.00              
ATOM      6  CG1 VAL A 171       6.749   1.178   2.942  1.00  0.00              
ATOM      7 HG11 VAL A 171       6.295   2.113   3.319  1.00  0.00              
ATOM      8 HG12 VAL A 171       7.396   0.780   3.747  1.00  0.00              
ATOM      9 HG13 VAL A 171       7.416   1.462   2.106  1.00  0.00              
ATOM     10  CG2 VAL A 171       6.395  -1.043   1.822  1.00  0.00              
ATOM     11 HG21 VAL A 171       5.675  -1.766   1.394  1.00  0.00              
ATOM     12 HG22 VAL A 171       7.039  -0.710   0.988  1.00  0.00              
ATOM     13 HG23 VAL A 171       7.038  -1.604   2.525  1.00  0.00              
ATOM     14  C   VAL A 171       3.253  -0.820   3.070  1.00  0.00              
ATOM     15  O   VAL A 171       3.216  -1.285   1.925  1.00  0.00              
ATOM     16  N   PRO A 172       2.105  -0.846   3.825  1.00  0.00              
ATOM     17  CD  PRO A 172       2.092  -0.629   5.289  1.00  0.00              
ATOM     18  CA  PRO A 172       0.825  -1.414   3.316  1.00  0.00              
ATOM     19  HA  PRO A 172       1.010  -2.196   2.551  1.00  0.00              
ATOM     20  CB  PRO A 172       0.302  -2.122   4.581  1.00  0.00              
ATOM     21  HB2 PRO A 172      -0.795  -2.268   4.579  1.00  0.00              
ATOM     22  HB3 PRO A 172       0.748  -3.135   4.648  1.00  0.00              
ATOM     23  CG  PRO A 172       0.785  -1.271   5.760  1.00  0.00              
ATOM     24  HG2 PRO A 172       0.047  -0.487   5.994  1.00  0.00              
ATOM     25  HG3 PRO A 172       0.915  -1.865   6.684  1.00  0.00              
ATOM     26  HD2 PRO A 172       2.134   0.448   5.545  1.00  0.00              
ATOM     27  HD3 PRO A 172       2.941  -1.128   5.793  1.00  0.00              
ATOM     28  C   PRO A 172      -0.156  -0.336   2.733  1.00  0.00              
ATOM     29  O   PRO A 172      -0.640   0.551   3.448  1.00  0.00              
ATOM     30  N   CYS A 173      -0.504  -0.453   1.438  1.00  0.00              
ATOM     31  H   CYS A 173       0.054  -1.103   0.868  1.00  0.00              
ATOM     32  CA  CYS A 173      -1.470   0.451   0.739  1.00  0.00              
ATOM     33  HA  CYS A 173      -1.093   1.476   0.737  1.00  0.00              
ATOM     34  CB  CYS A 173      -1.426   0.043  -0.740  1.00  0.00              
ATOM     35  HB2 CYS A 173      -0.934  -0.919  -0.902  1.00  0.00              
ATOM     36  HB3 CYS A 173      -2.445  -0.005  -1.108  1.00  0.00              
ATOM     37  SG  CYS A 173      -0.441   1.104  -1.803  1.00  0.00              
ATOM     38  C   CYS A 173      -2.904   0.570   1.371  1.00  0.00              
ATOM     39  O   CYS A 173      -3.497   1.653   1.322  1.00  0.00              
ATOM     40  N   SER A 174      -3.419  -0.497   2.014  1.00  0.00              
ATOM     41  H   SER A 174      -2.794  -1.312   2.014  1.00  0.00              
ATOM     42  CA  SER A 174      -4.651  -0.433   2.864  1.00  0.00              
ATOM     43  HA  SER A 174      -5.489  -0.123   2.235  1.00  0.00              
ATOM     44  CB  SER A 174      -4.935  -1.857   3.392  1.00  0.00              
ATOM     45  HB2 SER A 174      -5.909  -1.864   3.920  1.00  0.00              
ATOM     46  HB3 SER A 174      -5.052  -2.560   2.548  1.00  0.00              
ATOM     47  OG  SER A 174      -3.913  -2.329   4.278  1.00  0.00              
ATOM     48  HG  SER A 174      -3.981  -1.811   5.088  1.00  0.00              
ATOM     49  C   SER A 174      -4.687   0.582   4.057  1.00  0.00              
ATOM     50  O   SER A 174      -5.745   0.829   4.638  1.00  0.00              
ATOM     51  N   THR A 175      -3.529   1.130   4.427  1.00  0.00              
ATOM     52  H   THR A 175      -2.731   0.790   3.876  1.00  0.00              
ATOM     53  CA  THR A 175      -3.382   2.162   5.492  1.00  0.00              
ATOM     54  HA  THR A 175      -4.335   2.218   6.051  1.00  0.00              
ATOM     55  CB  THR A 175      -2.301   1.691   6.511  1.00  0.00              
ATOM     56  HB  THR A 175      -2.326   2.389   7.373  1.00  0.00              
ATOM     57  OG1 THR A 175      -0.991   1.740   5.950  1.00  0.00              
ATOM     58  HG1 THR A 175      -1.012   1.254   5.114  1.00  0.00              
ATOM     59  CG2 THR A 175      -2.533   0.264   7.050  1.00  0.00              
ATOM     60 HG21 THR A 175      -3.565   0.144   7.432  1.00  0.00              
ATOM     61 HG22 THR A 175      -2.412  -0.493   6.249  1.00  0.00              
ATOM     62 HG23 THR A 175      -1.828   0.005   7.854  1.00  0.00              
ATOM     63  C   THR A 175      -3.129   3.617   4.946  1.00  0.00              
ATOM     64  O   THR A 175      -3.370   4.586   5.667  1.00  0.00              
ATOM     65  N   CYS A 176      -2.649   3.785   3.693  1.00  0.00              
ATOM     66  H   CYS A 176      -2.727   2.956   3.097  1.00  0.00              
ATOM     67  CA  CYS A 176      -2.502   5.097   3.021  1.00  0.00              
ATOM     68  HA  CYS A 176      -2.028   5.819   3.715  1.00  0.00              
ATOM     69  CB  CYS A 176      -1.512   4.775   1.880  1.00  0.00              
ATOM     70  HB2 CYS A 176      -1.867   4.048   1.154  1.00  0.00              
ATOM     71  HB3 CYS A 176      -1.316   5.629   1.213  1.00  0.00              
ATOM     72  SG  CYS A 176       0.054   4.227   2.587  1.00  0.00              
ATOM     73  C   CYS A 176      -3.876   5.718   2.590  1.00  0.00              
ATOM     74  O   CYS A 176      -4.098   6.915   2.777  1.00  0.00              
ATOM     75  N   GLU A 177      -4.772   4.902   2.000  1.00  0.00              
ATOM     76  H   GLU A 177      -4.382   4.002   1.709  1.00  0.00              
ATOM     77  CA  GLU A 177      -6.192   5.240   1.695  1.00  0.00              
ATOM     78  HA  GLU A 177      -6.487   4.467   0.972  1.00  0.00              
ATOM     79  CB  GLU A 177      -7.062   5.042   2.978  1.00  0.00              
ATOM     80  HB2 GLU A 177      -6.712   5.755   3.751  1.00  0.00              
ATOM     81  HB3 GLU A 177      -8.102   5.362   2.770  1.00  0.00              
ATOM     82  CG  GLU A 177      -7.117   3.618   3.595  1.00  0.00              
ATOM     83  HG2 GLU A 177      -6.100   3.287   3.867  1.00  0.00              
ATOM     84  HG3 GLU A 177      -7.663   3.650   4.556  1.00  0.00              
ATOM     85  CD  GLU A 177      -7.747   2.542   2.713  1.00  0.00              
ATOM     86  OE1 GLU A 177      -7.145   1.939   1.830  1.00  0.00              
ATOM     87  OE2 GLU A 177      -9.057   2.329   3.009  1.00  0.00              
ATOM     88  C   GLU A 177      -6.497   6.563   0.931  1.00  0.00              
ATOM     89  O   GLU A 177      -7.380   7.352   1.278  1.00  0.00              
ATOM     90  N   GLY A 178      -5.771   6.738  -0.174  1.00  0.00              
ATOM     91  H   GLY A 178      -4.874   6.251  -0.096  1.00  0.00              
ATOM     92  CA  GLY A 178      -5.868   7.957  -1.000  1.00  0.00              
ATOM     93  HA2 GLY A 178      -5.863   7.613  -2.045  1.00  0.00              
ATOM     94  HA3 GLY A 178      -6.837   8.447  -0.854  1.00  0.00              
ATOM     95  C   GLY A 178      -4.756   9.018  -0.858  1.00  0.00              
ATOM     96  O   GLY A 178      -4.786  10.016  -1.578  1.00  0.00              
ATOM     97  N   ASN A 179      -3.762   8.813   0.028  1.00  0.00              
ATOM     98  H   ASN A 179      -3.681   7.838   0.332  1.00  0.00              
ATOM     99  CA  ASN A 179      -2.591   9.700   0.136  1.00  0.00              
ATOM    100  HA  ASN A 179      -2.837  10.724  -0.206  1.00  0.00              
ATOM    101  CB  ASN A 179      -1.987   9.611   1.550  1.00  0.00              
ATOM    102  HB2 ASN A 179      -2.755   9.462   2.335  1.00  0.00              
ATOM    103  HB3 ASN A 179      -1.355   8.699   1.535  1.00  0.00              
ATOM    104  CG  ASN A 179      -1.073  10.777   1.886  1.00  0.00              
ATOM    105  OD1 ASN A 179       0.084  10.831   1.486  1.00  0.00              
ATOM    106  ND2 ASN A 179      -1.572  11.753   2.588  1.00  0.00              
ATOM    107 HD21 ASN A 179      -2.553  11.656   2.863  1.00  0.00              
ATOM    108 HD22 ASN A 179      -0.920  12.520   2.759  1.00  0.00              
ATOM    109  C   ASN A 179      -1.580   9.147  -0.853  1.00  0.00              
ATOM    110  O   ASN A 179      -0.952   8.102  -0.646  1.00  0.00              
ATOM    111  N   LEU A 180      -1.410   9.875  -1.931  1.00  0.00              
ATOM    112  H   LEU A 180      -2.002  10.705  -2.021  1.00  0.00              
ATOM    113  CA  LEU A 180      -0.605   9.371  -3.032  1.00  0.00              
ATOM    114  HA  LEU A 180      -1.037   8.365  -3.112  1.00  0.00              
ATOM    115  CB  LEU A 180      -0.985  10.288  -4.200  1.00  0.00              
ATOM    116  HB2 LEU A 180      -1.109  11.331  -3.825  1.00  0.00              
ATOM    117  HB3 LEU A 180      -0.136  10.271  -4.849  1.00  0.00              
ATOM    118  CG  LEU A 180      -2.183   9.881  -5.065  1.00  0.00              
ATOM    119  HG  LEU A 180      -2.988   9.744  -4.346  1.00  0.00              
ATOM    120  CD1 LEU A 180      -2.568  11.005  -6.040  1.00  0.00              
ATOM    121 HD11 LEU A 180      -2.782  11.955  -5.515  1.00  0.00              
ATOM    122 HD12 LEU A 180      -1.766  11.213  -6.775  1.00  0.00              
ATOM    123 HD13 LEU A 180      -3.478  10.754  -6.616  1.00  0.00              
ATOM    124  CD2 LEU A 180      -1.966   8.576  -5.852  1.00  0.00              
ATOM    125 HD21 LEU A 180      -1.746   7.717  -5.193  1.00  0.00              
ATOM    126 HD22 LEU A 180      -2.861   8.298  -6.440  1.00  0.00              
ATOM    127 HD23 LEU A 180      -1.123   8.657  -6.566  1.00  0.00              
ATOM    128  C   LEU A 180       0.925   9.142  -2.807  1.00  0.00              
ATOM    129  O   LEU A 180       1.532   8.315  -3.481  1.00  0.00              
ATOM    130  N   ALA A 181       1.499   9.854  -1.841  1.00  0.00              
ATOM    131  H   ALA A 181       0.784  10.427  -1.396  1.00  0.00              
ATOM    132  CA  ALA A 181       2.858   9.591  -1.317  1.00  0.00              
ATOM    133  HA  ALA A 181       3.529   9.463  -2.179  1.00  0.00              
ATOM    134  CB  ALA A 181       3.308  10.827  -0.547  1.00  0.00              
ATOM    135  HB1 ALA A 181       3.303  11.721  -1.192  1.00  0.00              
ATOM    136  HB2 ALA A 181       2.656  11.008   0.328  1.00  0.00              
ATOM    137  HB3 ALA A 181       4.337  10.676  -0.173  1.00  0.00              
ATOM    138  C   ALA A 181       2.940   8.301  -0.431  1.00  0.00              
ATOM    139  O   ALA A 181       3.816   7.471  -0.661  1.00  0.00              
ATOM    140  N   CYS A 182       2.028   8.117   0.551  1.00  0.00              
ATOM    141  H   CYS A 182       1.210   8.729   0.442  1.00  0.00              
ATOM    142  CA  CYS A 182       1.879   6.835   1.307  1.00  0.00              
ATOM    143  HA  CYS A 182       2.729   6.596   1.912  1.00  0.00              
ATOM    144  CB  CYS A 182       0.811   7.117   2.355  1.00  0.00              
ATOM    145  HB2 CYS A 182       0.951   8.095   2.826  1.00  0.00              
ATOM    146  HB3 CYS A 182      -0.166   7.170   1.879  1.00  0.00              
ATOM    147  SG  CYS A 182       0.740   5.805   3.582  1.00  0.00              
ATOM    148  C   CYS A 182       1.661   5.588   0.393  1.00  0.00              
ATOM    149  O   CYS A 182       2.318   4.551   0.512  1.00  0.00              
ATOM    150  N   LEU A 183       0.750   5.765  -0.557  1.00  0.00              
ATOM    151  H   LEU A 183       0.216   6.641  -0.478  1.00  0.00              
ATOM    152  CA  LEU A 183       0.498   4.845  -1.669  1.00  0.00              
ATOM    153  HA  LEU A 183       0.242   3.877  -1.250  1.00  0.00              
ATOM    154  CB  LEU A 183      -0.734   5.465  -2.331  1.00  0.00              
ATOM    155  HB2 LEU A 183      -0.511   6.502  -2.606  1.00  0.00              
ATOM    156  HB3 LEU A 183      -0.863   5.000  -3.262  1.00  0.00              
ATOM    157  CG  LEU A 183      -2.105   5.338  -1.670  1.00  0.00              
ATOM    158  HG  LEU A 183      -1.989   5.895  -0.725  1.00  0.00              
ATOM    159  CD1 LEU A 183      -3.141   6.016  -2.574  1.00  0.00              
ATOM    160 HD11 LEU A 183      -3.118   5.603  -3.602  1.00  0.00              
ATOM    161 HD12 LEU A 183      -4.167   5.875  -2.202  1.00  0.00              
ATOM    162 HD13 LEU A 183      -2.957   7.102  -2.660  1.00  0.00              
ATOM    163  CD2 LEU A 183      -2.521   3.865  -1.469  1.00  0.00              
ATOM    164 HD21 LEU A 183      -1.797   3.296  -0.861  1.00  0.00              
ATOM    165 HD22 LEU A 183      -3.495   3.765  -0.960  1.00  0.00              
ATOM    166 HD23 LEU A 183      -2.589   3.318  -2.430  1.00  0.00              
ATOM    167  C   LEU A 183       1.672   4.591  -2.675  1.00  0.00              
ATOM    168  O   LEU A 183       1.775   3.499  -3.232  1.00  0.00              
ATOM    169  N   SER A 184       2.523   5.599  -2.902  1.00  0.00              
ATOM    170  H   SER A 184       2.300   6.383  -2.272  1.00  0.00              
ATOM    171  CA  SER A 184       3.804   5.430  -3.664  1.00  0.00              
ATOM    172  HA  SER A 184       3.590   4.875  -4.599  1.00  0.00              
ATOM    173  CB  SER A 184       4.373   6.807  -4.075  1.00  0.00              
ATOM    174  HB2 SER A 184       4.584   7.431  -3.185  1.00  0.00              
ATOM    175  HB3 SER A 184       5.351   6.674  -4.578  1.00  0.00              
ATOM    176  OG  SER A 184       3.500   7.497  -4.969  1.00  0.00              
ATOM    177  HG  SER A 184       2.688   7.688  -4.469  1.00  0.00              
ATOM    178  C   SER A 184       4.895   4.582  -2.911  1.00  0.00              
ATOM    179  O   SER A 184       5.744   3.967  -3.561  1.00  0.00              
ATOM    180  N   LEU A 185       4.895   4.564  -1.561  1.00  0.00              
ATOM    181  H   LEU A 185       4.052   4.999  -1.152  1.00  0.00              
ATOM    182  CA  LEU A 185       5.761   3.644  -0.759  1.00  0.00              
ATOM    183  HA  LEU A 185       6.760   3.563  -1.203  1.00  0.00              
ATOM    184  CB  LEU A 185       5.885   4.151   0.710  1.00  0.00              
ATOM    185  HB2 LEU A 185       4.910   3.993   1.210  1.00  0.00              
ATOM    186  HB3 LEU A 185       6.588   3.490   1.248  1.00  0.00              
ATOM    187  CG  LEU A 185       6.313   5.619   0.945  1.00  0.00              
ATOM    188  HG  LEU A 185       5.585   6.254   0.399  1.00  0.00              
ATOM    189  CD1 LEU A 185       6.231   5.974   2.438  1.00  0.00              
ATOM    190 HD11 LEU A 185       5.214   5.813   2.841  1.00  0.00              
ATOM    191 HD12 LEU A 185       6.927   5.369   3.050  1.00  0.00              
ATOM    192 HD13 LEU A 185       6.476   7.037   2.617  1.00  0.00              
ATOM    193  CD2 LEU A 185       7.728   5.917   0.423  1.00  0.00              
ATOM    194 HD21 LEU A 185       8.495   5.285   0.908  1.00  0.00              
ATOM    195 HD22 LEU A 185       7.802   5.744  -0.667  1.00  0.00              
ATOM    196 HD23 LEU A 185       8.014   6.971   0.592  1.00  0.00              
ATOM    197  C   LEU A 185       5.282   2.167  -0.818  1.00  0.00              
ATOM    198  O   LEU A 185       6.094   1.249  -0.977  1.00  0.00              
ATOM    199  N   CYS A 186       3.968   1.948  -0.682  1.00  0.00              
ATOM    200  H   CYS A 186       3.433   2.722  -0.295  1.00  0.00              
ATOM    201  CA  CYS A 186       3.376   0.633  -0.788  1.00  0.00              
ATOM    202  HA  CYS A 186       4.062  -0.005  -0.229  1.00  0.00              
ATOM    203  CB  CYS A 186       2.036   0.721  -0.053  1.00  0.00              
ATOM    204  HB2 CYS A 186       1.538  -0.234  -0.149  1.00  0.00              
ATOM    205  HB3 CYS A 186       2.029   0.919   1.013  1.00  0.00              
ATOM    206  SG  CYS A 186       0.974   1.935  -0.709  1.00  0.00              
ATOM    207  C   CYS A 186       3.166   0.020  -2.211  1.00  0.00              
ATOM    208  O   CYS A 186       3.204   0.641  -3.276  1.00  0.00              
ATOM    209  N   HIS A 187       2.862  -1.270  -2.130  1.00  0.00              
ATOM    210  H   HIS A 187       3.106  -1.635  -1.206  1.00  0.00              
ATOM    211  CA  HIS A 187       2.562  -2.149  -3.279  1.00  0.00              
ATOM    212  HA  HIS A 187       2.907  -1.685  -4.217  1.00  0.00              
ATOM    213  CB  HIS A 187       3.386  -3.455  -3.104  1.00  0.00              
ATOM    214  HB2 HIS A 187       2.974  -4.012  -2.270  1.00  0.00              
ATOM    215  HB3 HIS A 187       3.214  -4.096  -3.972  1.00  0.00              
ATOM    216  CG  HIS A 187       4.892  -3.322  -2.867  1.00  0.00              
ATOM    217  ND1 HIS A 187       5.401  -3.162  -1.588  1.00  0.00              
ATOM    218  CD2 HIS A 187       5.712  -2.703  -3.806  1.00  0.00              
ATOM    219  CE1 HIS A 187       6.468  -2.360  -1.880  1.00  0.00              
ATOM    220  NE2 HIS A 187       6.778  -2.075  -3.183  1.00  0.00              
ATOM    221  HD2 HIS A 187       5.312  -2.467  -4.775  1.00  0.00              
ATOM    222  HE1 HIS A 187       6.845  -1.738  -1.083  1.00  0.00              
ATOM    223  C   HIS A 187       1.023  -2.423  -3.366  1.00  0.00              
ATOM    224  O   HIS A 187       0.349  -2.654  -2.356  1.00  0.00              
ATOM    225  N   ILE A 188       0.477  -2.460  -4.588  1.00  0.00              
ATOM    226  H   ILE A 188       1.125  -2.221  -5.345  1.00  0.00              
ATOM    227  CA  ILE A 188      -0.987  -2.718  -4.819  1.00  0.00              
ATOM    228  HA  ILE A 188      -1.557  -2.448  -3.912  1.00  0.00              
ATOM    229  CB  ILE A 188      -1.529  -1.774  -5.956  1.00  0.00              
ATOM    230  HB  ILE A 188      -0.895  -1.991  -6.829  1.00  0.00              
ATOM    231  CG2 ILE A 188      -2.988  -2.071  -6.400  1.00  0.00              
ATOM    232 HG21 ILE A 188      -3.701  -1.986  -5.559  1.00  0.00              
ATOM    233 HG22 ILE A 188      -3.333  -1.384  -7.194  1.00  0.00              
ATOM    234 HG23 ILE A 188      -3.100  -3.094  -6.806  1.00  0.00              
ATOM    235  CG1 ILE A 188      -1.390  -0.242  -5.684  1.00  0.00              
ATOM    236 HG12 ILE A 188      -0.327   0.002  -5.491  1.00  0.00              
ATOM    237 HG13 ILE A 188      -1.624   0.316  -6.611  1.00  0.00              
ATOM    238  CD1 ILE A 188      -2.235   0.338  -4.533  1.00  0.00              
ATOM    239 HD11 ILE A 188      -2.042  -0.187  -3.581  1.00  0.00              
ATOM    240 HD12 ILE A 188      -2.008   1.408  -4.369  1.00  0.00              
ATOM    241 HD13 ILE A 188      -3.319   0.264  -4.735  1.00  0.00              
ATOM    242  C   ILE A 188      -1.265  -4.250  -5.078  1.00  0.00              
ATOM    243  O   ILE A 188      -2.207  -4.806  -4.511  1.00  0.00              
ATOM    244  N   GLU A 189      -0.481  -4.924  -5.941  1.00  0.00              
ATOM    245  H   GLU A 189       0.330  -4.399  -6.266  1.00  0.00              
ATOM    246  CA  GLU A 189      -0.592  -6.375  -6.219  1.00  0.00              
ATOM    247  HA  GLU A 189      -1.648  -6.552  -6.474  1.00  0.00              
ATOM    248  CB  GLU A 189       0.241  -6.703  -7.495  1.00  0.00              
ATOM    249  HB2 GLU A 189       0.035  -7.760  -7.753  1.00  0.00              
ATOM    250  HB3 GLU A 189      -0.178  -6.146  -8.357  1.00  0.00              
ATOM    251  CG  GLU A 189       1.790  -6.519  -7.463  1.00  0.00              
ATOM    252  HG2 GLU A 189       2.211  -6.915  -6.521  1.00  0.00              
ATOM    253  HG3 GLU A 189       2.243  -7.147  -8.253  1.00  0.00              
ATOM    254  CD  GLU A 189       2.303  -5.089  -7.652  1.00  0.00              
ATOM    255  OE1 GLU A 189       2.376  -4.257  -6.752  1.00  0.00              
ATOM    256  OE2 GLU A 189       2.663  -4.835  -8.938  1.00  0.00              
ATOM    257  C   GLU A 189      -0.278  -7.343  -5.032  1.00  0.00              
TER     258      GLU A 189 
END   
