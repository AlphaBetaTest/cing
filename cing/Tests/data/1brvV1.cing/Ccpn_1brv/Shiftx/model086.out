# Entries marked with a * may have inaccurate shift predictions.
# Entries marked with a value -666 should be igonored
# Output format adjusted by GV 11 September 2007
  171   V  N      114.0584
  171   V  CA      60.7267
  171   V  CB      32.5529
  171   V  C      172.8432
  171   V  H        7.8976
  171   V  HA       4.0387
  171   V  HB       2.1058
  171   V  HG11     0.9402
  171   V  HG12     0.9693
  171   V  HG13     0.6656
  171   V  HG21     0.9955
  171   V  HG22     1.0767
  171   V  HG23     0.9717
  172   P  CA      60.7116
  172   P  CB      32.4987
  172   P  C      176.9196
  172   P  HA       4.8040
  172   P  HB2      2.2891
  172   P  HB3      2.1864
  172   P  HD2      3.7069
  172   P  HD3      3.7728
  172   P  HG2      2.0049
  172   P  HG3      2.0499
  173   C  N      122.0493
  173   C  CA      59.8410
  173   C  CB      40.8247
  173   C  C      177.0775
  173   C  H        8.7772
  173   C  HA       4.2733
  173   C  HB2      3.0990
  173   C  HB3      3.1784
  174   S  N      115.8027
  174   S  CA      60.0557
  174   S  CB      62.4355
  174   S  C      175.3655
  174   S  H        8.4358
  174   S  HA       4.1745
  174   S  HB2      4.0158
  174   S  HB3      4.0094
  175   T  N      111.7835
  175   T  CA      62.7989
  175   T  CB      69.7836
  175   T  C      175.8523
  175   T  H        7.6077
  175   T  HA       4.3974
  175   T  HB       4.2387
  175   T  HG21     1.2098
  175   T  HG22     1.2099
  175   T  HG23     1.2102
  176   C  N      119.2928
  176   C  CA      58.6865
  176   C  CB      39.2783
  176   C  C      174.0522
  176   C  H        7.9123
  176   C  HA       4.3375
  176   C  HB2      3.1390
  176   C  HB3      3.2045
  177   E  N      118.2243
  177   E  CA      57.5753
  177   E  CB      26.5260
  177   E  C      175.8435
  177   E  H        9.5231
  177   E  HA       3.6111
  177   E  HB2      2.1072
  177   E  HB3      1.9616
  177   E  HG2      2.3748
  177   E  HG3      2.2245
  178   G  N      104.2609
  178   G  CA      45.1841
  178   G  C      173.4622
  178   G  H        8.3569
  178   G  HA    -666.0000
  178   G  HA2      3.9571
  178   G  HA3      3.9565
  179   N  N      119.2720
  179   N  CA      52.4376
  179   N  CB      39.3979
  179   N  C      175.3161
  179   N  H        7.9220
  179   N  HA       4.6012
  179   N  HB2      2.8077
  179   N  HB3      2.7231
  179   N  HD21     7.1386
  179   N  HD22     6.5432
  180   L  N      129.0736
  180   L  CA      57.2760
  180   L  CB      41.6338
  180   L  C      179.4310
  180   L  H        8.4613
  180   L  HA       3.9836
  180   L  HB2      1.7223
  180   L  HB3      1.6986
  180   L  HD11     0.9173
  180   L  HD12     0.9171
  180   L  HD13     0.9172
  180   L  HD21     0.2346
  180   L  HD22     1.6903
  180   L  HD23     0.8653
  180   L  HG       1.7174
  181   A  N      120.7761
  181   A  CA      55.1881
  181   A  CB      18.4122
  181   A  C      179.7029
  181   A  H        7.8994
  181   A  HA       3.9171
  181   A  HB1      1.3856
  181   A  HB2      1.3512
  181   A  HB3      1.3302
  182   C  N      116.6582
  182   C  CA      60.1205
  182   C  CB      43.4542
  182   C  C      175.9355
  182   C  H        7.9902
  182   C  HA       4.1072
  182   C  HB2      2.9980
  182   C  HB3      3.2254
  183   L  N      119.2264
  183   L  CA      57.8987
  183   L  CB      41.2065
  183   L  C      179.4767
  183   L  H        8.0060
  183   L  HA       3.8602
  183   L  HB2      1.8601
  183   L  HB3      1.7058
  183   L  HD11     0.9127
  183   L  HD12     0.9878
  183   L  HD13     0.9136
  183   L  HD21     0.4063
  183   L  HD22     1.6605
  183   L  HD23     0.8590
  183   L  HG       1.6623
  184   S  N      114.4054
  184   S  CA      61.1280
  184   S  CB      62.7428
  184   S  C      176.4224
  184   S  H        7.8350
  184   S  HA       4.2628
  184   S  HB2      4.0041
  184   S  HB3      3.9144
  185   L  N      121.5669
  185   L  CA      56.8282
  185   L  CB      41.6576
  185   L  C      175.3150
  185   L  H        7.5179
  185   L  HA       4.1533
  185   L  HB2      1.6738
  185   L  HB3      1.7106
  185   L  HD11     0.9136
  185   L  HD12     0.9170
  185   L  HD13     0.9160
  185   L  HD21     0.3198
  185   L  HD22     2.7126
  185   L  HD23     0.8668
  185   L  HG       1.7975
  186   C  N      111.0201
  186   C  CA      54.4993
  186   C  CB      42.7456
  186   C  C      172.0996
  186   C  H        7.1871
  186   C  HA       4.6456
  186   C  HB2      3.0499
  186   C  HB3      3.0517
  187   H  N      116.7795
  187   H  CA      54.3508
  187   H  CB      30.3023
  187   H  C      175.1585
  187   H  H        8.1107
  187   H  HA       4.9100
  187   H  HB2      3.0944
  187   H  HB3      3.4052
  187   H  HD2      5.8821
  187   H  HE1      6.9320
  188   I  N      125.8367
  188   I  CA      63.9604
  188   I  CB      37.8592
  188   I  C      176.6332
  188   I  H        7.9859
  188   I  HA       3.9026
  188   I  HB       1.8734
  188   I  HD11     0.8503
  188   I  HD12     0.8485
  188   I  HD13     0.8549
  188   I  HG12     1.1443
  188   I  HG13     0.7794
  188   I  HG21     0.9107
  188   I  HG22     0.9107
  188   I  HG23     0.9136
 *189   E  N      118.9195
 *189   E  CA      56.0770
 *189   E  CB      30.2537
 *189   E  C      177.0463
 *189   E  H        8.0232
 *189   E  HA       4.1609
 *189   E  HB2      1.7406
 *189   E  HB3      1.7729
 *189   E  HG2      2.1447
 *189   E  HG3      2.3848
